graphein/protein/config.py

```diff
@@ -33,7 +33,7 @@ class GetContactsConfig(BaseModel):
 class ProteinGraphConfig(BaseModel):
     """Config Object for Protein Structure Graph Construction"""
 
-    granularity: str = "CA"
+    granularity: Literal["atom", "CA", "CB", "centroid"] = "CA"
```

Comment:
In addition to these, we can also allow graph construction from any atoms.
Eg:
`['N', 'CA', 'C', 'O', 'CB', 'OG', 'CG', 'CD1', 'CD2', 'CE1', 'CE2',
       'CZ', 'OD1', 'ND2', 'CG1', 'CG2', 'CD', 'CE', 'NZ', 'OD2', 'OE1',
       'NE2', 'OE2', 'OH', 'NE', 'NH1', 'NH2', 'OG1', 'SD', 'ND1', 'SG',
       'NE1', 'CE3', 'CZ2', 'CZ3', 'CH2', 'OXT']`

I suppose we should add these to the literal?

Reply:
Added these to the literal as legibly as I could manage.